.github/workflows/PR_reviewer.yml

```
@@ -14,6 +14,7 @@ on:
 
 jobs:
   pr_reviewer_job:
+    if: github.event.pull_request.title != 'Deploy to Production'
```

I think we should maybe do this in javascript because there could be "release.." prs as well in the future, and it allows us to capture case insensitive etc

Example: 
```suggestion
    if: github.event.pull_request.title != 'Deploy to production'
```

I think most prs have lowercase p in production

---

I'll take a look on the script.